plugins/qcheck-stm/src/stm_of_ir.ml

```diff
@@ -460,10 +460,11 @@ let postcond_case config state idx state_ident new_state_ident value =
     in
     let pat_ret =
       match value.ret with
-      | None ->
+      | [] ->
           if may_raise_exception value then pvar (str_of_ident res_default)
           else ppat_any
-      | Some id -> pvar (str_of_ident id)
+      | [ id ] -> pvar (str_of_ident id)
+      | xs -> ppat_tuple (List.map (fun x -> pvar (str_of_ident x)) xs)
```

Comment:
This is currently deadcode, because `Ptyp_tuple` is not accepted in the match for `ret_ty` above, or did I miss something?
This makes it a bit harder to review. I think I would rather expect:
```ocaml
if may_raise_exception value then pvar (str_of_ident res_default)
else match value.ret with ...
```
because we don’t want to decompose right away the `result` of exception-raising functions.

If this is indeed dead code right now, it might make more sense to flag it explicitly as such, with a `failwith`, keeping all that code in a branch until we support tuples.